src/encoded/static/components/app.js

```diff
@@ -34,8 +34,11 @@ const portal = {
             title: 'Encyclopedia',
             children: [
                 { id: 'aboutannotations', title: 'About', url: '/data/annotations/' },
+                { id: 'sep-mm-1' },
```

Comment:
This is a React key. This mechanism’s been around for a very long time and was never commented. Should I comment it? I don’t mind, but I’ve gotten enough criticism the past few months for commenting things in detail that I’m a bit reticent.

Reply:
I think commenting is an art that will never please everyone. I will look this up as I have not seen it before. Since you say it has been used for a long time, it is safe to assume everyone should know it.